// andina/ma-accordion/fesm2020/ma-accordion.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class MaAccordionService {
    constructor() { }
}
MaAccordionService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
MaAccordionService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });

class MaAccordionComponent {
    constructor() { }
    ngOnInit() {
    }
}
MaAccordionComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
MaAccordionComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "14.3.0", type: MaAccordionComponent, selector: "lib-ma-accordion", ngImport: i0, template: `
    <p>
      ma-accordion works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionComponent, decorators: [{
            type: Component,
            args: [{ selector: 'lib-ma-accordion', template: `
    <p>
      ma-accordion works!
    </p>
  ` }]
        }], ctorParameters: function () { return []; } });

class MaAccordionModule {
}
MaAccordionModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
MaAccordionModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, declarations: [MaAccordionComponent], exports: [MaAccordionComponent] });
MaAccordionModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        MaAccordionComponent
                    ],
                    imports: [],
                    exports: [
                        MaAccordionComponent
                    ]
                }]
        }] });

/*
 * Public API Surface of ma-accordion
 */

/**
 * Generated bundle index. Do not edit.
 */

export { MaAccordionComponent, MaAccordionModule, MaAccordionService };
