// andina/ma-accordion/esm2020/lib/ma-accordion.module.mjs
import { NgModule } from '@angular/core';
import { MaAccordionComponent } from './ma-accordion.component';
import * as i0 from "@angular/core";
export class MaAccordionModule {
}
MaAccordionModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
MaAccordionModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, declarations: [MaAccordionComponent], exports: [MaAccordionComponent] });
MaAccordionModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.3.0", ngImport: i0, type: MaAccordionModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        MaAccordionComponent
                    ],
                    imports: [],
                    exports: [
                        MaAccordionComponent
                    ]
                }]
        }] });
